File: test-driven-development/vending-machine/c-sharp/tests/Vending.Machine.Test/VendingMachineTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System;
using System.Collections.Generic;
using System.Text;

namespace Vending.Machine.Test
{
    [TestClass]
    public class VendingMachineTest
    {
        private VendingMachine _machine;

        [TestInitialize]
        public void Setup()
        {
            _machine = new VendingMachine();
        }

        [TestMethod]
        public void WhenInsertingADimeThenBalanceIsTenCents()
        {
            _machine.InsertCoin("DIME");

            Assert.AreEqual("$0.10", _machine.DisplayText);
        }
    }
}

File: test-driven-development/vending-machine/java/src/test/java/vending/machine/VendingMachineTest.java
package vending.machine;

import org.junit.Test;

import static org.junit.Assert.*;

public class VendingMachineTest {
    @Test
    public void testSomething() {
        VendingMachine vendingMachine = new VendingMachine();
        assertEquals(0, vendingMachine.doSomething());
    }
}
